src/test/scala/scala/tools/refactoring/tests/util/TestHelper.scala

```diff
@@ -241,13 +262,29 @@ trait TestHelper extends TestRules with Refactoring with CompilerProvider with c
   }
 
   def selection(refactoring: Selections with InteractiveScalaCompiler, project: FileSet) = {
+    def containsError(t: refactoring.global.Tree): Boolean = {
+      def symbolContainsError = {
+        val sym = t.symbol
+        sym != null && {
```

Comment:
it was just proposal, if it is readible so LGTM too

2016-04-25 21:26 GMT+02:00 Matthias Langer <notifications@github.com>:

> In src/test/scala/scala/tools/refactoring/tests/util/TestHelper.scala
> <https://github.com/scala-ide/scala-refactoring/pull/160#discussion_r60972897>
> :
>
> > @@ -241,13 +262,29 @@ trait TestHelper extends TestRules with Refactoring with CompilerProvider with c
> >    }
> >
> >    def selection(refactoring: Selections with InteractiveScalaCompiler, project: FileSet) = {
> > +    def containsError(t: refactoring.global.Tree): Boolean = {
> > +      def symbolContainsError = {
> > +        val sym = t.symbol
> > +        sym != null && {
>
> I would leave it as is too... @wpopielarski
> <https://github.com/wpopielarski>?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/scala-ide/scala-refactoring/pull/160/files/2fc1b3b2a8044ca4a6a742082fe1b9466dc1511f#r60972897>
>